tests/Rector/Contrib/Nette/Application/InjectPropertyRector/InjectPropertyRectorTest.php

```diff
@@ -25,11 +27,8 @@ public function provideWrongToFixedFiles(): array
         ];
     }
 
-    /**
-     * @return string[]
-     */
-    protected function getRectorClasses(): array
+    protected function provideConfig(): string
     {
-        return [InjectPropertyRector::class];
+        return __DIR__ . '/config.yml';
```

Comment:
Isn't a simple `return config.yml` enough? All config files live in the same test directory, right?

Reply:
Relative paths can fail more often than absolute ones. Explicit over implicit.

Also better IDE supported with click right to the file

There is even fixer for that: https://github.com/Symplify/CodingStandard/blob/master/README.md#includerequire-should-be-followed-by-absolute-path